arviz/stats/stats.py

```diff
@@ -765,7 +753,40 @@ def hpd_ufunc(ary):
     metric_names.append("gelman-rubin statistic")
 
     joined = xr.concat(metrics, dim="metric").assign_coords(metric=metric_names)
-    return joined
+
+    if fmt.lower().startswith('wide'):
```

Comment:
Why starts with? If I read correctly fmt has to be exactly `"wide"`

Reply:
Yes, this is old feature.